### PR TITLE
feat: rename CHAIN_NAMESPACE.Bitcoin -> CHAIN_NAMESPACE.Utxo

### PR DESCRIPTION
**Pull request by gomesalexandre**
### Description

BREAKING CHANGE: CHAIN_NAMESPACE.Bitcoin is now CHAIN_NAMESPACE.Utxo

This renames CHAIN_NAMESPACE.Bitcoin to CHAIN_NAMESPACE.Utxo since it
now refers to any UTXO chain

### CODE REVIEW

**Comment by gomesalexandre** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1006** <a href="https://app.graphite.dev/github/pr/shapeshift/lib/1006" target="_blank">Graphite</a>  👈
    * **PR #1007** <a href="https://app.graphite.dev/github/pr/shapeshift/lib/1007" target="_blank">Graphite</a> 
      * **PR #1008** <a href="https://app.graphite.dev/github/pr/shapeshift/lib/1008" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/shapeshift/lib/1006?utm_source=stack-comment).

**Review by 0xApotheosis — APPROVED** [review]
I like it, though I'd much prefer Pascal case rather than all-caps for acronyms, i.e. `Utxo`.

**Review by 0xdef1cafe — CHANGES_REQUESTED** [review]
the bike shed has been painted - `PascalCase` or ["upper camel case"](https://rust-lang.github.io/api-guidelines/naming.html#casing-conforms-to-rfc-430-c-case) for enum members per the rusty people

**Review by 0xdef1cafe — APPROVED** [review]

**Comment by 0xApotheosis** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/shapeshift/lib/1006?utm_source=stack-comment).


Job Reference: 3Gb5nbnZH8bmUx8G1ASR

**Pull request merged**
